### DISCUSSION TITLE
User Registration on SQL server

### DISCUSSION THREAD

**Discussion by mserey** [elsa-workflows/elsa-core#598] [Q&A]
in the user registration example, how to change the store from mongo to sql server in the custom ativities. 
public CreateUser (SqlServerContext store, ...??
there is no AddSqlServerCollection for sql server
Please Help me

**Comment by wangadeketan** [comment]
Hi,
All you should have to do is copy the Elsa database tables (if you want to keep existing data) to your application's DB, and then the connection string used by Elsa to point to your application's database.

**Comment by TracyHZ** [comment]
I meet the same problem, please kindly guide it. Thanks!
